### DISCUSSION TITLE
Max Search Nodes?? Manager locking up

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12771] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I have 24 search nodes and the manager keeps locking up but I can't figure out why looking in the logs.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Can you explain` locking up` a bit more? Do containers exit?
